README.md

```diff
@@ -59,6 +61,12 @@ See [react-ui](https://github.com/CrossMint/crossmint-client-sdk/tree/main/packa
 
 ---
 
+## Migration guide to 0.1.0 version
```

Comment:
I would put this above "quick setup" for now, and then in a few weeks we bring it lower